deployment/k8s-config/harbor/values-keel-dev.yaml

```diff
@@ -1,42 +1,48 @@
 
-logLevel: info
-#logLevel: debug
+logLevel: debug
 
 externalURL: https://images-rc.canfar.net
 
-harborAdminPassword: changeme
-
 containerSecurityContext:
   runAsGroup: 1001
+  runAsUser: 0
   runAsNonRoot: true
```

Comment:
These two lines seem contradictory:  `runAsUser: 0` and `runAsNonRoot: true`.  Should `runAsUser: 0` be deleted?

Reply:
I removed the runAsUser declaration.